### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Angular-Neumorphism**

### CODE REVIEW

File: src/app/components/neo-card/neo-card/neo-card.component.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import "./card.theme";
+@import "../../neo-divider/neo-divider/neo-divider.component.scss";
```
Comment (vharadkou) [inline src/app/components/neo-card/neo-card/neo-card.component.scss:2]: card dependency?

File: src/app/components/neo-card/neo-card/neo-card.component.scss [hunk 2]
```diff
@@ -14,18 +15,19 @@ $neo-card-header-size: 40px !default;
...
     [dir="rtl"] & {
       left: auto;
-      right: 0;
+      right: 1%;
     }
 
     &.neo-divider-inset {
       position: static;
-      margin: 0;
+      width: calc(99% - #{$neo-divider-inset-margin});
```
Comment (vharadkou) [inline src/app/components/neo-card/neo-card/neo-card.component.scss:30]: why we need to calculate this property?

File: src/app/components/neo-card/neo-card/neo-card.component.scss [hunk 3]
```diff
@@ -14,18 +15,19 @@ $neo-card-header-size: 40px !default;
...
   .neo-divider-horizontal {
+    --divider-offset: $neo-divider-inset-margin;
     position: absolute;
     left: 1%;
     width: 98%;
 
     [dir="rtl"] & {
       left: auto;
-      right: 0;
+      right: 1%;
```
Comment (vharadkou) [inline src/app/components/neo-card/neo-card/neo-card.component.scss:25]: in %?

File: src/app/components/neo-card/neo-card/neo-card.component.scss [hunk 4]
```diff
@@ -14,18 +15,19 @@ $neo-card-header-size: 40px !default;
   border-radius: $neo-card-border-radius;
 
   .neo-divider-horizontal {
+    --divider-offset: $neo-divider-inset-margin;
```
Comment (vharadkou) [inline src/app/components/neo-card/neo-card/neo-card.component.scss:18]: why we need to use it?

File: src/app/conteiners/cardpage/cardpage.component.html [hunk 5]
```diff
@@ -1,30 +1,70 @@
...
+    </neo-card-footer>
+  </neo-card>
 
-<neo-card class="card">
-  <neo-card-header>
-    <neo-card-title>Basic card</neo-card-title>
-  </neo-card-header>
-</neo-card>
+  <code class="code">
+    <pre>
```
Comment (vharadkou) [inline src/app/conteiners/cardpage/cardpage.component.html:31]: just for info: in future need to create separate component for it

File: src/app/components/neo-card/neo-card/neo-card.component.scss (hunk 1 above)
Comment (Angular-Neumorphism) [reply src/app/components/neo-card/neo-card/neo-card.component.scss:2]: Card redefines some divider props inside self

**Pull request merged**